setup.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import print_function
```

Comment:
Awesome!

I landed on jeff's site from the audrey's blog(author for the cookiecutter module).

I checked the setup.py[cookiecutter module](https://github.com/audreyr/cookiecutter) and it looked pretty good.

I highly suggest trying it out if you haven't already.